Note about shared mailbox (#893)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeHybrid/hybrid-deployment/move-mailboxes.md

3. On the **Select a migration type** page, select **Remote move migration** and then click **Next**.
    
4. On the **Select the users** page, click **Add**![Add Icon](../media/ITPro_EAC_AddIcon.gif) and select the on-premises users to move to Office 365 and click **Add** and then click **OK**. Click **Next**.

> [!NOTE]

>     > If shared mailbox doesn't appear in Users selections list, you need to sync Shared mailbox AD account first to Office365 Using Azure AD Connect after sync completed, they will appear in portal as blocked account and they will be available in Users selections list

Reviewer: ```suggestion
>  If a shared mailbox doesn't appear in the user selection list, you will first need to sync the shared mailbox Azure AD account to Office 365 using Azure AD Connect. The shared mailbox will then appear as a blocked account that is available for selection.
```

Author: If a shared mailboxes doesn't appear in the user selection list, you will first need to sync Shared mailboxes On-promise AD accounts  to Office 365 using Azure AD Connect. The shared Mailboxes AD accounts  will appear as a blocked accounts in Office 365 portal after that they will be available  in users selection list .

> If shared mailbox accounts don't appear in the user selection list, you need to sync the shared mailbox on-premises AD accounts to Office 365 using Azure AD Connect. The shared mailbox AD accounts will appear as blocked accounts in the Office 365 portal and you will be able to select them from the user list.```

Reviewer: ```suggestion
If shared mailbox accounts don't appear in the user selection list, you need to sync the shared mailbox on-premises Azure AD accounts to Office 365 using Azure AD Connect. The shared mailbox  accounts will appear as blocked accounts in the Office 365 portal and you will be able to select them from the user list.```
```

> >  if a shared mailboxes doesn't appear in the user selection list, you will first need to sync Shared mailboxes On-promise AD accounts to Office 365 using Azure AD Connect. The shared Mailboxes AD accounts will appear as a blocked accounts in Office 365 portal after that they will be available in users selection list.

Reviewer: ```suggestion
> If shared mailbox accounts don't appear in the user selection list, you need to sync the shared mailbox on-premises AD accounts to Office 365 using Azure AD Connect. The shared mailbox AD accounts will appear as blocked accounts in the Office 365 portal and you will able to select them from the user list.
```

> >  if a shared mailboxes doesn't appear in the user selection list, you will first need to sync Shared mailboxes On-promise AD accounts to Office 365 using Azure AD Connect. The shared Mailboxes AD accounts will appear as a blocked accounts in Office 365 portal after that they will be available in users selection list.

Reviewer 2: ```suggestion
If shared mailbox accounts don't appear in the user selection list, you need to sync the shared mailbox on-premises AD accounts to Office 365 using Azure AD Connect. The shared mailbox AD accounts will appear as blocked accounts in the Office 365 portal and you will be able to select them from the user list.```
